docs/Video Streaming for Navigation Apps/Audio Streaming/index.md

```diff
@@ -40,7 +101,26 @@ sdlManager.streamManager?.audioManager.playNextWhenReady()
 sdlManager.streamManager?.audioManager.push(with: audioData)
 sdlManager.streamManager?.audioManager.playNextWhenReady()
 ```
+!@
+
+@![android]
+You can also send `ByteBuffer`s to the `AudioStreamManager` to be played. To use it, replace the `pushResource` call in the example above to the `pushBuffer` call shown below:
```

Comment:
I think this line might be able to go away to align with iOS above? Or should iOS add a line?